sw/device/lib/dif/BUILD

```diff
@@ -300,12 +301,15 @@ cc_library(
 cc_library(
     name = "hmac",
     srcs = [
+        "autogen/dif_hmac_autogen.c",
+        "autogen/dif_hmac_autogen.h",
         "dif_hmac.c",
     ],
     hdrs = [
         "dif_hmac.h",
     ],
     deps = [
+        ":base",
         "//hw/ip/hmac/data:hmac_regs",
         "//sw/device/lib/base",
     ],
```

Comment:
Autogen'd unit tests have been created and merged (#8304) into master so you could add a unit test target for HMAC too.

Reply:
I got that one working. Are there any others that are autogen'd I should be running?